compiler/frontend/src/builtin_functions.rs

```diff
@@ -36,6 +36,9 @@ pub enum BuiltinFunction {
     StructGet,           // struct key -> value
     StructGetKeys,       // struct -> listOfKeys
     StructHasKey,        // struct key -> booleanSymbol
+    TagGetSymbol,        // symbol -> symbol
```

Comment:
To the user, I wouldn't even mention the word symbol and instead, use tags consistently (which either have values or don't). This builtin could then be named `tagWithoutValue` (or something like that).

I'd also rename the other references here (for example, change the comment of `structHasKey` to `struct key -> booleanTag`).

Of course, the rest of our internal compiler code would continue to use "symbol" like it already does.